app/vite.config.js

```diff
@@ -4,4 +4,8 @@ import react from '@vitejs/plugin-react'
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  base: './',
+  build: {
+    target: "ES2022",
```

Comment:
ES2022 is added because DuckDB.tsx have a "outer level await", which is not supported in some older version.